src/main/scala/com/atomist/rug/runtime/js/JavaScriptCommandHandler.scala

```diff
@@ -103,7 +104,10 @@ class JavaScriptCommandHandler(jsc: JavaScriptContext,
   override def handle(ctx: RugContext, params: ParameterValues): Option[Plan] = {
     val validated = addDefaultParameterValues(params)
     validateParameters(validated)
-    invokeMemberFunction(jsc, handler, "handle", ctx, validated) match {
+    // We need to proxy the context to allow for property access
+    invokeMemberFunction(jsc, handler, "handle",
+      jsScalaHidingProxy(ctx, returnNotToProxy = jsSafeCommittingProxy.DoNotProxy),
```

Comment:
We proxy this now to allow property access. We must be sure not to proxy some methods, hence the function passed in.